wagtail/admin/templates/wagtailadmin/pages/edit.html

```diff
@@ -13,7 +13,11 @@
         <div class="row row-flush">
             <div class="left col9 header-title">
                 <h1 class="icon icon-doc-empty-inverse">
-                {% blocktrans with title=page.get_admin_display_title page_type=content_type.model_class.get_verbose_name %}Editing {{ page_type }} <span>{{ title }}</span>{% endblocktrans %}</h1>
+                    {% blocktrans with title=page.get_admin_display_title page_type=content_type.model_class.get_verbose_name %}Editing {{ page_type }} <span>{{ title }}</span>{% endblocktrans %}
```

Comment:
This combination of page type + page name + help text could get pretty long, and doesn't look great when it wraps as far as I can tell when testing quickly - is there any way we could look at the wrapping here, or put the help text below instead?

Reply:
Hey @jacobtoppm I'm going to remove this until the new design work is in core as to prevent design conflicts.